doc/api/util.md

```diff
@@ -630,7 +630,7 @@ const util = require('util');
 
 const o = {
   a: [1, 2, [[
-    'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do ' +
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed \ndo ' +
```

Comment:
Is the `\n` supposed to be the last thing in the string on this line?

Reply:
I updated the example further to outline the functionality stronger and also fixed a few more outdated documentation parts.